GridTariffAPI.lib/Services/TariffQueryService.cs

```diff
@@ -261,7 +273,14 @@ public virtual MeteringPointsAndPriceLevels MeteringPointsToPriceLevel(string fi
             }
         }
 
-        private async Task<SeasonDataAccumulator> ProcessSeasonAsync(TariffPrice tariffPrice, Models.PriceStructure.TariffPrice tariffPricePrice, Models.PriceStructure.TariffType tariffType, List<Holiday> filteredHolidays, Models.PriceStructure.Season season, TimePeriod seasonIntersect)
+        private async Task<SeasonDataAccumulator> ProcessSeasonAsync(
+            TariffPrice tariffPrice, 
+            Models.PriceStructure.TariffPrice tariffPricePrice, 
+            Models.PriceStructure.TariffType tariffType, 
+            List<Holiday> filteredHolidays, 
+            Models.PriceStructure.Season season, 
+            TimePeriod seasonIntersect,
+            FixedPrices fixedpPrices)
```

Comment:
En p for mye i variabelnavnet

Reply:
Addresserer typo.
Selv rettelsen er committet her: https://github.com/3lvia/grid-tariff-api/pull/109/commits/cd5c77b07b8da825dbf4f29c370f525ae4ec121d

Valgte imidlertid i etterkant å sende en parameter som eksplisitt parameter til funksjon fremfor at funksjonen henter parameter fra liste med LastOrDefault(). 
(Det gjorde at det ble flere endringer enn strengt tatt nødvendig, og litt rotete i PR.